### DISCUSSION TITLE
Какая-то проблема с check.js и build\index.js

### DISCUSSION THREAD

**Discussion by ff278** [MrZillaGold/VK2Discord#654] [Q&A] [answered]
я не понимаю в чем проблема, возможно я что-то не скачал, возможно что-то упустил, но вот записи ошибок
1. npm start
2. node .
![image](https://user-images.githubusercontent.com/70628254/196807151-18d8f0ed-f4b6-4cd0-b1a6-8f6431733979.png)

**Comment by egorprnn** [comment, marked as answer]
Сборку скрипта необходимо скачивать из [релизов](https://github.com/MrZillaGold/VK2Discord/releases). Сейчас можете выполнить npm run-script build и запустить скрипт заново.

**Reply by ff278** [reply]
![image](https://user-images.githubusercontent.com/70628254/197343951-6b603842-fe2a-4634-929f-67cb0931b83e.png)
сначала без npm i, но потом скачало и пишет что есть какие-то уязвимости, и при запуске такая же ошибка

**Reply by ff278** [reply]
также вел команду которая советует npm audit fix, такая же пишет ошибка при ошибке, при том что скачал последний релиз

**Reply by egorprnn** [reply]
Ошибки после npm i можно игнорировать, они не влияют на работу

**Reply by ff278** [reply]
ну а так, все время вылезает ошибка
возможно проблема в node она у меня 19.0.0

**Reply by ff278** [reply]
а вот на другом пк с 18.х.х запускается нормально ни на что не жалуется
